### ISSUE TITLE
make all traps and planned moat visible in spectatormode

### ISSUE THREAD

**Issue by Monsterfisch** [Sh0wdown/UnofficialCrusaderPatch#619]
Make planned moats, pitch traps and murder holes of all ai's visible to the spectating player similar to the way that assassins are visible now.

**Comment by J-T-de** [comment]
Depending on how it is implemented, it might be easily possible to adapt this feature for Multiplayer, which is really bad. I find this really problematic.

**Comment by Monsterfisch** [comment]
The same thing would be true with the assassins and so far that one has not been abused in that sort of way. consider this would be spectator mode exclusive

**Comment by J-T-de** [comment]
How can you know that people didn't abuse visible assassions online?

**Comment by Monsterfisch** [comment]
I guess you wanna take out the assassin thing then?

**Comment by J-T-de** [comment]
No, I don't because it is too late now^^

I don't say we should not do this (and I am not someone who decides this), I really think this is a nice addition for spectator mode, but I see it potentially be abused. I don't care about multiplayer, but others do, and for them it might be a big no-no. (Recently, some UCP developers got offers to create such cheats for Multiplayer,at least shows the interest in such cheats).

Just for reference, the old post regarding the assassins: https://github.com/Sh0wdown/UnofficialCrusaderPatch/issues/408.

**Comment by Monsterfisch** [comment]
yeah I am aware of the requests to get cheaty business going on but that would happen anyway if it's possible at all if capable people are convinced of it being a worthwhile endeavor this sort of feature wouldn't even really even send out more data than it already does consider the way the game handles multiplayer.

**Comment by Monsterfisch** [comment]
yeah i am aware of those offers xP but that's just a natural thing when people cant win the normal way :D
